src/components/manage/Blocks/Maps/MapsData.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { MapsSchema } from './schema';
+import { BlockDataForm } from '@plone/volto/components';
+import { useIntl, defineMessages } from 'react-intl';
+import globeSVG from '@plone/volto/icons/globe.svg';
+import { Icon } from '@plone/volto/components';
+import { Segment } from 'semantic-ui-react';
+
+const messages = defineMessages({
+  Maps: {
+    id: 'Maps',
+    defaultMessage: 'Maps',
+  },
+  NoMaps: {
+    id: 'No map selected',
+    defaultMessage: 'No map selected',
+  },
+});
+
+const MapsData = (props) => {
+  const { data, block, onChangeBlock, schemaEnhancer } = props;
+  const intl = useIntl();
+  const schema = schemaEnhancer
```

Comment:
Do you have a use case where you have to pass down schemaEnhancer as a prop? The reason is that there's an automatic schemaEnhancer step done in the BlockDataForm component.

Reply:
ok! I haven't look into it. I will remove it if it is already done.

Reply:
removed it.